doc/devel/documenting_mpl.rst

```diff
@@ -54,9 +54,17 @@ requirements that are needed to build the documentation. They are listed in
 
 .. note::
 
-  * You'll need a minimal working LaTeX distribution for many examples to run.
+  * You will need a minimal working LaTeX and Graphviz distribution for many
+    examples to run.
   * `Graphviz <http://www.graphviz.org/Download.php>`_ is not a Python package,
     and needs to be installed separately.
+  * Create a seprate environment using conda/venv.
+  * You will need to install the addtional requirements using pip install -r.
```

Comment:
Document which files to `-r` from?

Reply:
@timhoffm Need some help here.

Reply:
`pip install -r requirements/doc/doc-requirements.txt`

My text above is just a rough outline. I think all commands should be explicitly written out, so that the user just needs to copy the respective commands.